### PR TITLE
Rename to TracingClientInterceptor and TracingServerInterceptor

### PR DESCRIPTION
**Pull request by ravirajj**
- Rename `ClientTracingInterceptor` to `TracingClientInterceptor`.
- Rename `ServerTracingInterceptor` to `TracingServerInterceptor`.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -3,6 +3,7 @@ dist: trusty
 
 language: java
 jdk:
+  - openjdk8
```
Comment (malafeev) [inline .travis.yml]: I expect it breaks release process because it's completely done by travis and now two streams will make a release.

Comment (ravirajj) [reply .travis.yml]: Travis supports testing against multiple JDKs: https://docs.travis-ci.com/user/languages/java#testing-against-multiple-jdks.  

It passes build.  Did you still want me to revert this?

Comment (malafeev) [reply .travis.yml]: build is passed, until `release:prepare` will be triggered
look at `travis/publish.sh` script:
```
# If we are on a release tag, the following will update any version references and push a version tag for deployment.
elif build_started_by_tag; then
  safe_checkout_master
  ./mvnw --batch-mode -s ./.settings.xml -Prelease -nsu -DreleaseVersion="$(release_version)" -Darguments="-DskipTests" release:prepare
fi
```

It needs to be reverted or fixed to run only on oraclejdk8

Comment (ravirajj) [reply .travis.yml]: Done.

**Pull request merged**